source/Pages/best_practices.rst

```diff
@@ -0,0 +1,160 @@
+Feature                                          CephFS           dCache           Scratch         
+==============================================   ==============   ==============   =============   
+High throughput & Low load on the system         No               Yes              Yes             
+Large volumes of data                            No               Yes              Somewhat              
+Data available after jobs end                    Yes              Yes              No              
+Data available outside Spider                    No*              Yes              No              
+Granular access control                          Yes              Yes              No              
+Supports disk                                    Yes              Yes              Yes             
+Supports tape                                    No               Yes              No              
+Available through an API                         No               Yes              No                   
+==============================================   ==============   ==============   =============   
+
+* *unless explicitly placed in public folder*
+
+Which data storage practice should I use?
+-----------------------------------------
+
+For your bulk data storage we suggest you to use dCache. dCache is highly connected to Spider worker nodes and is designed for high-throughput processing of data. This storage system is also available outside of Spider, and has highly granular access controls, making data releases, or data uploader roles self service. dCache is also available through a number of interfaces, meaning that it can be used out of the box with webdav clients or through a REST API, allowing for future data portals to be developed. Another reason to use dCache is that it supports both disk and tape, meaning that it can easily scale to much more data. Here you can find instructions for using the :ref:`dCache remote storage <using-dcache>`.
+
+We also advice you to use the scratch file systems as fast temporary storage that can be used while running a job. Each of the Spider worker nodes has a large scratch area on local SSD. Any data that you wish to keep should be written to other storage backends such as dCache. The scratch areas are ideal for retrieving the input of a job from dCache during execution or for applications that generate lots of intermediate files that are consumed by other parts of the processing or for generating the job output before copying it back to dCache. More about scratch can be found in our section :ref:`How to use the temporary disk space <scratch-fs>`.
+
+In cases that you have multiple jobs that need to access a single set of files that is too large to copy over to scratch, it is possible to use CephFS on home or project space locations for temporary storing your data, but take into account its limitations such as slow throughput and short capacity compared to dCache.
```

Comment:
Shall we say "but take into account its limitations compared to dCache in terms of throughput and capacity"?  Phrases like "slow throughput and short capacity might give a negative impression of CephFS.

Reply:
Good point. Fixed.